doc/source/howtos/upgrade_guide.rst

```diff
@@ -21,50 +21,50 @@ and variables mentioned in these documents when upgrading to |PXC| 8.0.
-you can perform a `rolling upgrade from PXC 5.6 to 5.7
-<https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html>`_.
-and then follow the current procedure to upgrade from 5.7 to 8.0.
-
-.. rubric:: *Not* recommended to mix |pxc| 5.7 nodes with |pxc| 8.0 nodes
-
-If downtime is acceptable, shut down the cluster and upgrade all nodes to |pxc|
-8.0. It is important that you make backups before attempting an upgrade.
-
-If downtime is not possible, the rolling upgrade is supported but ensure the
-traffic is controlled during the upgrade and writes are directed only to 5.7
-nodes until all nodes are upgraded to 8.0.
-
-.. rubric:: The configuration file layout has changed in |pxc| |version|
-
-All configuration settings are stored in the default |MySQL| configuration file:
-:file:`my.cnf`:
-
-* Path on Debian and Ubuntu: :file:`/etc/mysql/my.cnf`
-* Path on Red Hat and CentOS: :file:`/etc/my.cnf`
-
-Before you start the upgrade, move your custom settings from
-:file:`/etc/mysql/percona-xtradb-cluster.conf.d/wsrep.cnf` (on Debian and
-Ubuntu) or from :file:`/etc/percona-xtradb-cluster.conf.d/wsrep.cnf` (on Red Hat
-and CentOS) to the new location accordingly.
-
-.. rubric:: ``caching_sha2_password`` Is the Default Authentication Plugin
-
-In |PXC| |version|, the default authentication plugin is
-``caching_sha2_password``. The ProxySQL option
-:ref:`pxc.proxysql.v2.admin-tool.syncusers` will not work if the |PXC| user is
-created using ``caching_sha2_password``. Use the ``mysql_native_password``
-authentication plugin in these cases.
-
+Traffic encryption enabled by default
+   The :variable:`pxc_encrypt_cluster_traffic` variable, which enables traffic
+   encryption, is set to ``ON`` by default in |pxc| |version|. For more
+   information, see :ref:`encrypt-traffic`.
+
+Rolling upgrades to |version| from versions older than 5.7 are not supported
+   Therefore, if you are running |PXC| version 5.6, shut down all nodes, then
+   remove and re-create the cluster from scratch. Alternatively, you can perform
+   a `rolling upgrade from PXC 5.6 to 5.7
+   <https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html>`_.
+   and then follow the current procedure to upgrade from 5.7 to 8.0.
+
+*Not* recommended to mix |pxc| 5.7 nodes with |pxc| 8.0 nodes
+   If downtime is acceptable, shut down the cluster and upgrade all nodes to
+   |pxc| 8.0. It is important that you make backups before attempting an
+   upgrade.
+
+   If downtime is not possible, the rolling upgrade is supported but ensure the
+   traffic is controlled during the upgrade and writes are directed only to 5.7
+   nodes until all nodes are upgraded to 8.0.
+
+The configuration file layout has changed in |pxc| |version|
+   All configuration settings are stored in the default |MySQL| configuration
+   file:
+
+   * Path on Debian and Ubuntu: :file:`/etc/mysql/mysql.conf.d/mysqld.cnf`
```

Comment:
On debian/ubuntu the file is `/etc/mysql/my.cnf` which then includes files in directory `/etc/mysql/mysql.conf.d` which has file `/etc/mysql/mysql.conf.d/mysqld.cnf` which in the end has all the new default options.
It's still not 3 files like before, but users should always check /etc/mysql/my.cnf and start from there since that is the main one and could include the wrong config/subdirectory.
I see your point that settings are actually in `/etc/mysql/mysql.conf.d/mysqld.cnf`, but it would be good to mention that the main file is `/etc/mysql/my.cnf` since from what I see in major upgrade that one can include wrong subdirectory.